packages/perennial/contracts/incentivizer/Incentivizer.sol

```diff
@@ -54,19 +55,19 @@ contract Incentivizer is IIncentivizer, UInitializable, UControllerProvider, URe
         ProgramInfoLib.validate(programInfo);
 
         // Take fee
-        (ProgramInfo memory newProgramInfo, UFixed18 programFeeAmount) = ProgramInfoLib.deductFee(programInfo, _controller.incentivizationFee());
-        fees[newProgramInfo.token] = fees[newProgramInfo.token].add(programFeeAmount);
+        UFixed18 programFeeAmount = programInfo.deductFee(_controller.incentivizationFee());
+        fees[programInfo.token] = fees[programInfo.token].add(programFeeAmount);
```

Comment:
could one line this one to: `fees[programInfo.token] = fees[programInfo.token].add(programInfo.deductFee(_controller.incentivizationFee()));`

Reply:
I think we need the `programFeeAmount` for the event at the bottom